src/en/help-azure.md

```diff
@@ -128,157 +144,163 @@ In the sections below, we will assign each of these a variable name.  When you
 
 List your account. Note the subscription ID, the **SUB_ID**.
 
 ```bash
-azure account list
+az account list
 ```
 
-**SUB_ID** will appear on a line like this:
+**SUB_ID** will appear on the line labelled **id**:
 
-```no-highlight
-info:    Executing command account list
-data:    Name        Id                                    Current  State
-data:    ----------  ------------------------------------  -------  -------
-data:    Free Trial  f717c8c1-8e5e-4d38-be7f-ed1e1c879e18  true     Enabled
-info:    account list command OK
+```yaml
+[
+  {
+    "cloudName": "AzureCloud",
+    "id": "f717c8c1-8e5e-4d38-be7f-ed1e1c879e18",
+    "isDefault": true,
+    "name": "Pay-As-You-Go",
+    "state": "Enabled",
+    "tenantId": "0fb95fd9-f42f-4c78-94c9-e3d01c2bc5af",
+    "user": {
+      "name": "javierlarin72@gmail.com",
+      "type": "user"
+    }
+  }
+]
 ```
 
-In the output of this command, the **SUB_ID** is not labeled as such. In our
-sample it was next to last line, so:
+In our sample, **SUB_ID** is the second line line, so:
 
 ```bash
 SUB_ID=f717c8c1-8e5e-4d38-be7f-ed1e1c879e18
 ```
 
 ### `application-password` and  `application-id`
 
-Create a password for the application to use, the **APP_PASSWORD**. In our
-sample,
+Create a password for the application to use. In our sample:
 
 ```bash
 APP_PASSWORD=some_password
 ```
 
-Create an Azure Active Directory (AAD) application:
+Now create an Active Directory (Kerberos) server principal:
 
 ```bash
-azure ad app create \
-        --name "ubuntu.example.com" \
-        --home-page "http://ubuntu.example.com" \
-        --identifier-uris "http://ubuntu.example.com" \
-        --password $APP_PASSWORD
+az ad sp create-for-rbac --name "ubuntu.example.com" --password $APP_PASSWORD
 ```
 
-The options `--name`, `--home-page`, and `--identifier-uris` are arbitrary but
-you should use values that make sense for your environment.
-
-In the output of this command, note the application ID, the **APP_ID**.
-In our sample it was on a line like this:
+The `--name` option is arbitrary but you should use a unique value that makes
+sense for your environment. The command output will be similar to the
+following:
 
-```bash
-data:    AppId:    f6ab7cbd-5029-43ef-85e3-5c4442a00ba8
+```yaml
+{
+  "appId": "01dfe0e9-f088-4d00-9fcf-2129de64d5d3",
+  "displayName": "ubuntu.example.com",
+  "name": "http://ubuntu.example.com",
+  "password": "$APP_PASSWORD",
+  "tenant": "0fb95fd9-f42f-4c78-94c9-e3d01c2bc5af"
+}
 ```
 
-Use the APP_ID to create an Active Directory (Kerberos) server principal:
+We'll be using the value that follows **appId** as **APP_ID** and **tenant** as
+**TENANT_ID** (used later). Also make a note of `Object ID`, which we'll refer
+to as **OBJ_ID**
 
-!!! Note: Replace our variable here with the actual value you learned above.
-Do this throughout the rest of this page when you see variables listed.
+Now grant permissions to the principal with the following command, replacing
```

Comment:
Pretty sure you can fold this into the `create-for-rbac` command with the --role flag.